goss-testing/scripts/check_bios_firmware_versions.sh

```diff
@@ -57,8 +57,38 @@ function set_vars() {
   fi
 
   # Allow for output of documentation when needed
+  rc=0
+
+  DL325_FW="2.46"
```

Comment:
Would we be better off reading these in from a file, so we wouldn't have to edit the script just to change expected versions?

Reply:
Yes, but I wasn't going to go down the road of ad-hoc config management when the larger discussion is still going on--just rolling with what's already in place here.